modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSetContext.tsx

```diff
@@ -14,7 +14,10 @@ export interface IFrontendDataSetContext {
 	applyItemInlineUpdates: Function;
 	createInlineItem: Function;
 	customDataRenderers?: Array<any>;
-	customRenderers?: {tableCell?: Array<TRenderer>};
+	customRenderers?: {
+		tableCell?: Array<TRenderer>;
+		views?: Array<TRenderer>;
+	};
```

Comment:
Updates types, based on https://github.com/liferay-frontend/liferay-portal/pull/4571#discussion_r1850709870

I also changed `view` to `views`. It's not consistent with `tableCell`, but it pained me too much to have an array called with singular :sweat_smile: